react-accordion/my-app/src/Accordion.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+import { TopicContent } from './TopicContent';
+import { TopicHeader } from './TopicHeader';
+
+type Topic = {
+  id: number;
+  title: string;
+  content: string;
+};
+type Props = {
+  topics: Topic[];
+};
+export function Accordion({ topics }: Props) {
+  const [displayId, setDisplayId] = useState<number>();
+
+  const topicBoxArray: JSX.Element[] = [];
+
+  for (let i = 0; i < topics.length; i++) {
+    const header = (
+      <TopicHeader
```

Comment:
I would find the code more logical if a Topic component contained both the header and the content.